aws/components/s3/setup.ftl

```diff
@@ -203,21 +202,11 @@
                     [/#if]
                     [#break]
 
+                [#case "baselinedata"]
```

Comment:
Better to use the component type Variable instead of fixed value. Shows the meaning of what this is a bit more